docs/source/credentials.rst

```diff
@@ -0,0 +1,39 @@
+Credentials
+***********
+
+Integration with Git
+====================
+
+Git and DataLad can use each other's credential system.
+In order to allow Git to query credentials from DataLad, the git credential helper delivered with DataLad needs to be configured.
+That is, a section like this needs to be part of one's git config file::
+
+  [credential "https://*.data.example.com"]
+    helper = "datalad"
```

Comment:
> This surprises me. The command datalad is enough to act as a git credential helper? I don't understand how that would work.

See `man gitcredentials` (value string does not begin with an absolute path). It's not a command. It results in calling `git-credential-datalad`. 

> Also, this example assumes that datalad is always in the path, which might not be the case with scripts, cron, etc.

At least the helper (technically not datalad itself) has to be in the path, yes. Otherwise one would need to give the absolute path to that helper. Not good for an example, as it may depend on what system you installed DataLad (and how). So, I wouldn't suggest any absolute path here.

Reply:
> See man gitcredentials (value string does not begin with an absolute path). It's not a command. It results in calling git-credential-datalad.

Ahh yeah, I remember reading that and disliking it. Now I am reminded why. ;-)